src/cli/commands/cryptoFund.ts

```diff
@@ -54,7 +55,7 @@ export async function cryptoFund(options: CryptoFundOptions) {
     const { winc } = await turbo.getWincForToken({ tokenAmount });
     const targetWallet = (await turbo.getTurboCryptoWallets())[token];
 
-    const credits = (+winc / 1_000_000_000_000).toFixed(12);
+    const credits = (+winc / wincPerCredit).toFixed(12);
```

Comment:
just changing var here, but fwiw this doesn't seem safe. we should keep winc as a bigNumber here to do the division